Sources/Plugins/Plugin.swift

```diff
@@ -1,18 +1,31 @@
 //
 //  Copyright © 2017 Nikita Ermolenko. All rights reserved.
+//  Copyright © 2019 Rosberry. All rights reserved.
 //
 
 import UIKit
 
  public protocol Plugin {
-    
-    var nib: UINib { get }
-    var cellIdentifier: String { get }
+    var cellClass: BaseTableViewCell.Type { get }
+
+    var title: NSAttributedString { get }
+    var description: NSAttributedString? { get }
+    var keywords: String { get }
 
-    func configure(_ cell: UITableViewCell)
     func selectionAction()
 }
 
 public extension Plugin {
-    func selectionAction() {}
+
+    var description: NSAttributedString? {
+        return nil
+    }
+
+    var keywords: String {
+        return title.string + (description?.string ?? "")
+    }
+
+    func selectionAction() {
+        //
```

Comment:
Why you leave empty comment here?

Reply:
Just a placeholder, it's a 'me' thing, it looks better in my opinion